noxfile.py

```diff
@@ -0,0 +1,120 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import nox
+
+TEST_DEPENDENCIES = [
+    "certifi",
+    "flask",
+    "mock",
+    "oauth2client",
+    "pytest",
+    "pytest-cov",
+    "pytest-localserver",
+    "requests",
+    "requests-oauthlib",
+    "urllib3",
+    "cryptography",
```

Comment:
Some of these seem required for the actual library to function. Those should be defined in `setup.py`, not here, right?

Reply:
This was a straight copy from the [tox.ini](https://github.com/googleapis/google-auth-library-python/blob/master/tox.ini). Some of them might not be needed anymore though. I'll go through and check